app/src/main/java/org/mifos/selfserviceapp/presenters/TransferProcessPresenter.java

```diff
@@ -0,0 +1,65 @@
+package org.mifos.selfserviceapp.presenters;
+
+import android.content.Context;
+
+import org.mifos.selfserviceapp.api.DataManager;
+import org.mifos.selfserviceapp.injection.ActivityContext;
+import org.mifos.selfserviceapp.models.payload.TransferPayload;
+import org.mifos.selfserviceapp.presenters.base.BasePresenter;
+import org.mifos.selfserviceapp.ui.views.TransferProcessView;
+import org.mifos.selfserviceapp.utils.MFErrorParser;
+
+import javax.inject.Inject;
+
+import okhttp3.ResponseBody;
+import rx.Subscriber;
+import rx.android.schedulers.AndroidSchedulers;
+import rx.schedulers.Schedulers;
+import rx.subscriptions.CompositeSubscription;
+
+/**
+ * Created by dilpreet on 1/7/17.
+ */
+
+public class TransferProcessPresenter extends BasePresenter<TransferProcessView> {
```

Comment:
@dilpreet96 there is no attach and detach implemented, It can be Observer leakage then It will be hard to find.